bot.py

```diff
@@ -201,7 +195,7 @@ async def _respond(msg):
     if teams is []:
         await _respond("You are not in a team")
         return
-    
+
```

Comment:
Half of this commit can be split into another one fixing whitespace / code style.